TestRapport.md

```diff
@@ -0,0 +1,11 @@
+# Test Rapport
+
+### Inleiding
+Voor het bepalen van verschillenden testen is het bepalen van de strategie de eerste stap. Hierbij kan worden gekozen uit een brede variatie aan strategieën.  
+Voor dit project zijn een aantal populaire strategieën gebruikt als leidraad. Zoals Big-bang, Top-down, Bottom-up, Sandwich en Modified Sandwich.
+
+### Strategie
+Bij de big bang strategie worden eerst individuele modules getest en vervolgens als een groot geheel. 
+Wij, 2nd-Factor hebben de Big bang strategie als leidraad gekozen. Dit doordat het product 2nd-Factor als geheel binnen de marges blijft om goed en volledig 
```

Comment:
```suggestion
Wij, 2nd-Factor hebben de Big bang strategie als leidraad gekozen. Dit doordat het product 2nd-Factor als geheel binnen de marges blijft om goed en volledig 
```

Is het niet ook omdat het een relatief klein project is dat in 1x wordt opgeleverd?